.github/workflows/bitcoin-tests.yml

```diff
@@ -59,7 +60,6 @@ jobs:
           - tests::neon_integrations::antientropy_integration_test
           - tests::neon_integrations::filter_low_fee_tx_integration_test
           - tests::neon_integrations::filter_long_runtime_tx_integration_test
-          - tests::neon_integrations::mining_transactions_is_fair
```

Comment:
Yup; it was removed in `master` because mining isn't "fair" anymore -- if you pay higher fee rates than the next person, your transactions are always prioritized over theirs.